Update mulled_containers.rst (#20446)
galaxyproject/galaxy: doc/source/admin/special_topics/mulled_containers.rst

This approach has shown two limitations that slowed down the adoption by tool developers.
First, every tool needs to be annotated with a container name (as shown above) and this container needs

> to be created beforehand, usually manually. The second reason is that a Galaxy tool aims to be deployed everywhere,

Reviewer: ```suggestion
to be created beforehand, usually manually. The second reason is that Galaxy tools should be deployable everywhere,
```

> independent of the underlying system, meaning if Docker is not available Galaxy should use Conda packages. 

Reviewer: ```suggestion
regardless of the underlying system, meaning that if Docker is not available, Galaxy should use Conda packages instead. 
```
